client/src/plugins/http/interceptors.js

```diff
@@ -0,0 +1,40 @@
+import { isArray } from 'lodash'
+
+export default (http, store, router) => {
+  // https://github.com/mzabriskie/axios#interceptors
+  http.interceptors.response.use(
+    response => response,
+    /**
+    * This is a central point to handle all
+    * error messages generated by HTTP
+    * requests
+    */
+    (error) => {
+      const { response } = error
+      /**
+      * If token is either expired, not provided or invalid
+      * then redirect to login. On server side the error
+      * messages can be changed on app/Providers/EventServiceProvider.php
+      */
+      if (response.status === 401) {
```

Comment:
Redirect to login when the status is also 400

response.status === 401  || === 400